app/css/app.css

```diff
@@ -1,52 +1,20 @@
+/** General styles **/
 html {
+  height: 100%;
+
   font: normal 10px/10px Arial;
 }
 
 body {
+  height: 100%;
   margin: 0;
   padding: 0;
+
+  background: #FFF;
```

Comment:
Btw, I really like https://google.github.io/styleguide/htmlcssguide.xml + BEM naming + properties grouping (https://github.com/necolas/idiomatic-css#declaration-order), not sure what you think.